core/plannernotes.c

```diff
@@ -268,13 +295,29 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 				/* Normally a gas change is displayed on the stopping segment, so only display a gas change at the end of
 				 * an ascent segment if it is not followed by a stop
 				 */
+
+				if(istrimix) {
+					deltaN2 = ((1000-newgasmix.he.permille-newgasmix.o2.permille)-(1000-lastprintgasmix.he.permille-lastprintgasmix.o2.permille))/10.0;
+					deltaHe = (newgasmix.he.permille-lastprintgasmix.he.permille)/10.0;
+					maxdeltaN2 = 0.02*(lastprintgasmix.he.permille-newgasmix.he.permille);
+					runtime = (dp->time + 30) / 60;
+					deltaPN2 = (1+((dp->depth.mm)/10000)) * deltaN2 / 100;
+					colorc = (deltaN2>maxdeltaN2)?strcpy(colorc,"red"):strcpy(colorc,"black");
+				}
+
 				if ((isascent || dp->entered) && gaschange_after && dp->next && nextdp && (dp->depth.mm != nextdp->depth.mm || nextdp->entered)) {
 					if (dp->setpoint) {
 						snprintf(temp, sz_temp, translate("gettextFromC", "(SP = %.1fbar)"), (double) nextdp->setpoint / 1000.0);
 						len += snprintf(buffer + len, sz_buffer - len, "<td style='padding-left: 10px; color: red; float: left;'><b>%s %s</b></td>", gasname(&newgasmix),
 									temp);
 					} else {
-							len += snprintf(buffer + len, sz_buffer - len, "<td style='padding-left: 10px; color: red; float: left;'><b>%s</b></td>", gasname(&newgasmix));
```

Comment:
You are correctly fixing a couple of old indentation mistakes here. Maybe that should go into a separate (first) commit.